dev-client/src/auth/index.ts

```diff
@@ -0,0 +1,46 @@
+import Config from 'react-native-config';
```

Comment:
suggestion: can we create a `config.ts` file like in the web client instead of directly accessing the env variables?

Reply:
OK, I'll try that out.